mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -6925,3 +7264,828 @@ resource "google_compute_disk" "debian" {
 }
 `, instance, diskName, suffix, suffix, suffix)
 }
+
+<% unless version == 'ga' -%>
+// The tests related to security_policy use network_edge_security_service resource
+// which can only exist one per region. Because of that, all the following tests must run serially.
+func TestAccComputeInstanceNetworkIntefaceWithSecurityPolicy(t *testing.T) {
+	testCases := map[string]func(t *testing.T){
+		"two_access_config": TestAccComputeInstance_nic_securityPolicyCreateWithTwoAccessConfigs,
```

Comment:
The first letter of the test names should be lower case. Otherwise, they are still run as separate tests. When running test, every test starting with `TestAcc` will be run. For example, `TestAccComputeInstance_nic_securityPolicyCreateWithTwoAccessConfigs` should be `testAccComputeInstance_nic_securityPolicyCreateWithTwoAccessConfigs`